package.json

```diff
@@ -156,51 +156,51 @@
     "chrome-launcher": "^0.12.0",
     "compression-webpack-plugin": "^3.1.0",
     "copy-webpack-plugin": "^5.1.1",
-    "cypress": "3.8.1",
-    "cypress-multi-reporters": "^1.2.3",
-    "depcheck": "^0.9.1",
+    "cypress": "4.0.2",
```

Comment:
We downgraded the cypress version as a measure to see if it helped the test run time. I can't find the issue to bump it back up... Can we check and see if this impacts the running time of some of the e2es?

Reply:
> We downgraded the cypress version as a measure to see if it helped the test run time. I can't find the issue to bump it back up... Can we check and see if this impacts the running time of some of the e2es?

Good point, completely forgot about that. Will remove it for now 👍

Reply:
Will also create an issue for investigating the new version of Cypress

Reply:
Issue for updating Cypress: https://github.com/bbc/simorgh/issues/5601